hubblestack/version.py

```diff
@@ -0,0 +1,4 @@
+# coding: utf-8
+
+__version__ = '4.0.0'
```

Comment:
Please move this file out to root folder in Hubble

Reply:
no